### PR TITLE
fix: Adjust visibility of missed reference members

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the new behavior?

Adjust the reference comparer tool to include member visibility on the implementation side.

### <samp>🤖 Generated by Copilot at c88c936</samp>

This pull request updates the reference assemblies for the Uno Platform to match the runtime assemblies and the WinUI API. It fixes some formatting and comparison issues, removes some unimplemented or Uno-specific members, and changes some constructors and methods from public to protected or private. It affects the files `PackageDiffIgnore.xml`, `Program.cs`, `FrameworkElement.skia.cs`, `ImageSource.netstd.cs`, `ImageSource.netstdref.cs`, `VibrationDevice.cs`, `Accelerometer.skia.cs`, `Accelerometer.others.cs`, and several unsupported sensor classes.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by ajpinedam — APPROVED** [review]

File: src/Uno.UWP/Generated/3.0.0.0/Windows.Devices.Sensors/Pedometer.cs [hunk 1]
```diff
@@ -79,7 +79,7 @@ public  double PowerInMilliwatts
 			throw new global::System.NotImplementedException("The member IAsyncOperation<Pedometer> Pedometer.FromIdAsync(string deviceId) is not implemented. For more information, visit https://aka.platform.uno/notimplemented?m=IAsyncOperation%3CPedometer%3E%20Pedometer.FromIdAsync%28string%20deviceId%29");
 		}
 		#endif
-		#if false || false || false || false || __SKIA__ || __NETSTD_REFERENCE__ || false
+		#if false || false || false || false || __SKIA__ || false || false
 		[global::Uno.NotImplemented("__SKIA__", "__NETSTD_REFERENCE__")]
```
Comment (Youssef1313) [inline src/Uno.UWP/Generated/3.0.0.0/Windows.Devices.Sensors/Pedometer.cs:83]:
```suggestion
		[global::Uno.NotImplemented("__SKIA__")]
```

File: src/Uno.UWP/Generated/3.0.0.0/Windows.Devices.Sensors/Magnetometer.cs [hunk 2]
```diff
@@ -122,7 +122,7 @@ public static string GetDeviceSelector()
 			throw new global::System.NotImplementedException("The member IAsyncOperation<Magnetometer> Magnetometer.FromIdAsync(string deviceId) is not implemented. For more information, visit https://aka.platform.uno/notimplemented?m=IAsyncOperation%3CMagnetometer%3E%20Magnetometer.FromIdAsync%28string%20deviceId%29");
 		}
 		#endif
-		#if false || false || false || false || __SKIA__ || __NETSTD_REFERENCE__ || false
+		#if false || false || false || false || __SKIA__ || false || false
 		[global::Uno.NotImplemented("__SKIA__", "__NETSTD_REFERENCE__")]
```
Comment (Youssef1313) [inline src/Uno.UWP/Generated/3.0.0.0/Windows.Devices.Sensors/Magnetometer.cs:126]:
```suggestion
		[global::Uno.NotImplemented("__SKIA__")]
```

File: src/Uno.UI/UI/Xaml/FrameworkElement.skia.cs [hunk 3]
```diff
@@ -71,7 +71,7 @@ private void OnNameChanged(string oldValue, string newValue)
 		}
 
 		[GeneratedDependencyProperty(DefaultValue = "", ChangedCallback = true)]
-		internal static DependencyProperty NameProperty { get; } = CreateNameProperty();
+		public static DependencyProperty NameProperty { get; } = CreateNameProperty();
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/FrameworkElement.skia.cs:74]: Likely my bad when moving the property from UIElement to FrameworkElement.

**Pull request merged**